project/plugins.sbt

```diff
@@ -1,7 +1,7 @@
 addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat"    % "0.1.20")
 addSbtPlugin("org.scalameta"             % "sbt-scalafmt"    % "2.4.6")
 addSbtPlugin("com.typesafe"              % "sbt-mima-plugin" % "1.0.1")
-addSbtPlugin("org.scoverage"             % "sbt-scoverage"   % "1.6.1")
+addSbtPlugin("org.scoverage"             % "sbt-scoverage"   % "1.9.3")
```

Comment:
SBT complains about this key:

```
arn]       lazy value coverageMinimum in object ScoverageKeys is deprecated (since v1.8.0): Use coverageMinimumStmtTotal instead
[warn]       L87:     coverageMinimum := 50,
```

Let's fix it in this commit as well.